src/Nav.php

```diff
@@ -252,7 +246,7 @@ private function renderItem(array $item): string
         if (!isset($item['label'])) {
             throw new InvalidArgumentException('The "label" option is required.');
         }
-
+        $this->dropdown = false;
```

Comment:
@art009 can we pass it to the method instead of using class property?

Reply:
Delete property.